app/renderer/components/Session/SavedSessions.js

```diff
@@ -31,6 +30,18 @@ export default class SavedSessions extends Component {
     return capsUUID === record.key ? SessionCSS.selected : '';
   }
 
+  handleCapsAndServer (session) {
+    const {setCapsAndServer, server, serverType } = this.props;
+
+    // Incase user has CAPS saved from older version of Inspector which
+    // doesn't contain server and serverType within the session object
+    if (('server' in session) && ('serverType' in session)) {
```

Comment:
more often we use the idiom `if (session.server)`; if it's undefined, null, or an empty string it will evaluate to false, which is often what we want.